docker/Dockerfile

```
@@ -175,10 +182,12 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     ca-certificates
 
-# Config settings
-
+# Config settings for zebrad
 ARG NETWORK
-ENV NETWORK=${NETWORK:-Mainnet}
+ENV NETWORK=${NETWORK}
+
+ARG FEATURES
+ENV FEATURES=${FEATURES}
 
```

We need these environmental variables to diagnose bugs in production images as well.

```suggestion

ARG RUST_BACKTRACE
ENV RUST_BACKTRACE=${RUST_BACKTRACE:-1}

ARG RUST_LIB_BACKTRACE
ENV RUST_LIB_BACKTRACE=${RUST_LIB_BACKTRACE:-1}

ARG COLORBT_SHOW_HIDDEN
ENV COLORBT_SHOW_HIDDEN=${COLORBT_SHOW_HIDDEN:-1}
```

---

To have these environment variables in production, is not required to have those in the Dockerfile. We can set these defaults (if needed) in the `entrypoint` script

---

Great! However you want to do it is fine with me.